dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/apps/kafka/permissions.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<permissions xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+            http://xmlns.jcp.org/xml/ns/javaee/permissions_7.xsd"
+        version="7">
+        
+<!--      <permission> -->
+<!--        <class-name>javax.management.MBeanServerPermission</class-name> -->
+<!--        <name>createMBeanServer</name> -->
+<!--      </permission> -->
+<!--      <permission> -->
+<!--        <class-name>javax.management.MBeanPermission</class-name> -->
+<!--        <name>*</name> -->
+<!--        <actions>*</actions> -->
+<!--      </permission> -->
+     
+<!--      <permission> -->
+<!--        <class-name>java.util.PropertyPermission</class-name> -->
+<!--        <name>*</name> -->
+<!--        <actions>read</actions> -->
+<!--      </permission> -->
+     
+    <permission>
```

Comment:
We should remove the AllPermission entry

Reply:
Done